src/index.md

```diff
@@ -3,7 +3,11 @@ layout: home
 page_class: home
 title: "Home"
 ---
```

Comment:
Let's add a line break between the content and the frontmatter block to ensure nothings gets broken